src/index.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter, Switch, Route } from "react-router-dom";
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Switch>
+        <Route path="/">
+            <App />
+        </Route>
+      </Switch>
+    </BrowserRouter>
```

Comment:
I know I've gone back-and-forth with you on this, but one final comment - I would make `index.js` just 

```jsx
<BrowserRouter>
  <App />
</BrowserRouter>
```

And then place the Switch and Route logic inside `App.js` - that way, it'll let you handle some of the more complex routing logic within a stateful component (`index.js` isn't actually a React component). Let me know if that makes sense/if you have any thoughts!

Reply:
Ah okay. What does it mean to be a stateful component?

Also, do I have to put the switch route statements in something else? When I put it in the return for App.js it just leads to the page eternally loading.

Reply:
By stateful component, I mean that you can treat `App.js` (or any component that gets imported into `index.js`) as a fully-fledged React component. However, `index.js` itself is not a component - in particular, that means that you can't run React-like code in `index.js`.

What I'd suggest is:

1. Rename what is now `App.js` to something else, since it isn't the entire app - it's just one page in the eventual learning lab
2. Now, move the `<Switch>` and `<Route>` stuff to a new file called `App.js`. This new `App.js` should also import the file you made in step 1
3. In `index.js`, just import the `App.js` and wrap it in a `BrowserRouter`, as I mentioned in the previous comment

This setup lets you use React logic in deciding what route to render and/or what props to pass down to each rendered component, which I think you'll find useful later on. If you're looking for an example, you can check out Passworks, which uses the structure I described above: [here's the `index.js`](https://github.com/uclaacm/passworks/blob/master/src/index.js), and [here's the `App.js`](https://github.com/uclaacm/passworks/blob/master/src/App.js).

Reply:
I see. That makes a lot of sense! I am recommitting to this branch with these changes.